packages/radix-vue/src/Calendar/CalendarNext.vue

```diff
@@ -19,7 +19,7 @@ const rootContext = injectCalendarRootContext()
 
 <template>
   <Primitive
-    v-bind="props"
+    :as="props.as"
```

Comment:
`PrimitiveProps` has `as`, and `asChild`. Let's add that too

Reply:
Thanks for the comment. I fixed the PR in https://github.com/radix-vue/radix-vue/pull/929/commits/cc7e33a719c8702853ff3c151ec0e3fb700987b9. 🙇